static/app/views/performance/cache/cacheLandingPage.tsx

```diff
@@ -10,33 +10,55 @@ import {EnvironmentPageFilter} from 'sentry/components/organizations/environment
 import PageFilterBar from 'sentry/components/organizations/pageFilterBar';
 import {ProjectPageFilter} from 'sentry/components/organizations/projectPageFilter';
 import {t} from 'sentry/locale';
+import {decodeScalar, decodeSorts} from 'sentry/utils/queryString';
 import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
 import {normalizeUrl} from 'sentry/utils/withDomainRequired';
 import {CacheHitMissChart} from 'sentry/views/performance/cache/charts/hitMissChart';
 import {ThroughputChart} from 'sentry/views/performance/cache/charts/throughputChart';
 import {Referrer} from 'sentry/views/performance/cache/referrers';
-import {MODULE_TITLE, RELEASE_LEVEL} from 'sentry/views/performance/cache/settings';
+import CacheSamplePanel from 'sentry/views/performance/cache/samplePanel/samplePanel';
+import {
+  BASE_FILTERS,
+  CACHE_BASE_URL,
+  MODULE_TITLE,
+  RELEASE_LEVEL,
+} from 'sentry/views/performance/cache/settings';
+import {
+  isAValidSort,
+  TransactionsTable,
+} from 'sentry/views/performance/cache/tables/transactionsTable';
 import {convertHitRateToMissRate} from 'sentry/views/performance/cache/utils';
 import * as ModuleLayout from 'sentry/views/performance/moduleLayout';
 import {ModulePageProviders} from 'sentry/views/performance/modulePageProviders';
+import {useSpanMetrics} from 'sentry/views/starfish/queries/useSpanMetrics';
 import {useSpanMetricsSeries} from 'sentry/views/starfish/queries/useSpanMetricsSeries';
-import type {SpanMetricsQueryFilters} from 'sentry/views/starfish/types';
+import {QueryParameterNames} from 'sentry/views/starfish/views/queryParameters';
+
+const DEFAULT_SORT = {
+  field: 'time_spent_percentage()' as const,
+  kind: 'desc' as const,
+};
+
+const TRANSACTIONS_TABLE_ROW_COUNT = 20;
```

Comment:
Nit: You can move these to the end of the file, so the constants don't distract from getting to the component declaration, which is the important part

Reply:
I love this nit, I totally agree, I typically prefer the main stuff at the top, and all the supporting stuff at the bottom. Lemme make that change